api/v1alpha08/kogitoserverlessplatform_types.go

```diff
@@ -86,6 +87,13 @@ type KogitoServerlessPlatformSpec struct {
 	BuildPlatform api.PlatformBuildSpec `json:"platform,omitempty"`
 	// Configuration list of configuration properties to be attached to all the Workflow built from this Platform
 	Configuration ConfigurationSpec `json:"configuration,omitempty"`
+	// Trait list of traits properties attached to all the Workflow built from this Platform
+	Traits Traits `json:"traits,omitempty"`
```

Comment:
shouldn't Traits be a list of Trait?

Reply:
Traits was used plural in the case of camel because of the number of traits they had underneath the Traits struct. Looking at it now, it makes sense to be singular so I agree Traits should be a list of Trait.

Reply:
We will have multiple traits soon, so it is more useful to maintain the plural (see [KOGITO-8245](https://issues.redhat.com/browse/KOGITO-8255))